_field-types/supported-field-types/star-tree.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: Star Tree
+nav_order: 61
+has_children: false
+parent: Supported field types
+redirect_from:
+  - /opensearch/supported-field-types/star-tree/
+  - /field-types/star-tree/
+---
+# Star tree field type
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+Star Tree Index (STIX) pre-computes aggregations to accelerate the performance of aggregation queries.
```

Comment:
thinking if there is way to avoid aggregations twice in the same sentence

Reply:
All the limits and defaults are mentioned in the documentation already [ which comes from StarTreeIndexSettings]. The cluster and index setting is mentioned separately.

Is there any setting that user can change which is not mentioned here ?

Reply:
We can wait for doc team review/feedback on this